source/reference/program/mongostat.txt

```diff
@@ -179,15 +179,21 @@ average operations per second.
    megabytes. This is the total data size at the time of the last
    :program:`mongostat` call.
 
-.. describe:: size
+.. describe:: vsize
 
    The amount of virtual memory in megabytes used by the process at
    the time of the last :program:`mongostat` call.
 
 .. describe:: non-mapped
 
-   The total amount of virtual memory excluding all mapped memory at
+   .. versionchanged:: 3.0.0 
+   Only for :ref:`storage-mmapv1`.
```

Comment:
I fixed the Debian step here, since it wasn't building at all for me either.

As for tag-aware-sharding, I moved it, but it should be fine (https://github.com/mongodb/docs/commit/6d9e8f2acbab065aa8244776b47614a9fc782970)? I just copy-pasted it across. Maybe I'll cherry-pick this commit into a new master.